docs/system_overview/federation.md

```diff
@@ -161,3 +161,65 @@ In order to support commands such as `ils` and `ilsresc` across a 3.x to 4.0+ fe
 
 The effect of turning this negotiation off is a lack of SSL encryption when talking with a 3.x zone.  All clients that connect to this 4.0+ zone will also need to disable the Advanced Negotiation in their own 'irods_environment.json' files.
 
+### How to enable federation between 3.x and 4.x:
+
+The steps required for enabling federation between a 3.x zone and a 4.x zone will be described through an example.
+
+Let ZoneA have the following properties:
+~~~
+host name: zoneA.example.org
+zone name: ZoneA
+zone key : ZONE_KEY_FOR_ALL_OF_ZONE_A
+zone port: 1247
+~~~
+and ZoneB have the following properties:
+~~~
+host name: zoneB.example.org
+zone name: ZoneB
+zone server ID : ZONEB_LOCAL_ZONE_SID
+zone port: 1247
+~~~
+The zone server ID (SID) is similar to a `zone_key` in 4.x.
+
+#### Configuring 4.x zone
+The `federation` stanza needs to be added to the `server_config.json` as usual:
+~~~
+"federation": [
+    {
+        "catalog_provider_hosts": ["zoneB.example.org"], 
+        "negotiation_key": "_____32_byte_pre_shared_key_____",
+        "zone_key": "ZONEB_LOCAL_ZONE_SID", 
+        "zone_name": "ZoneB", 
+        "zone_port": 1247
+    }
+]
+~~~
+Note: `negotiation_key` has no effect, but is required by the server configuration json schema.
+The `zone_key` member of the struct held by `federation` in this case acts as the `RemoteZoneSID` and must match the `LocalZoneSID` configured in the 3.x server.
+
+#### Configuring 3.x zone
+The 3.x zone only requires two things in its `server.config`:
+~~~
+LocalZoneSID ZONEB_LOCAL_ZONE_SID
+RemoteZoneSID ZoneA-ZONE_KEY_FOR_ALL_OF_ZONE_A
+~~~
+The `LocalZoneSID` is equivalent to the `zone_key` shared in 4.x federated environments.
+The `RemoteZoneSID` is a string of the form `remoteZoneName-remoteZoneSID`. In this example, `remoteZoneName` is `ZoneA` and the `remoteZoneSID` is the `zone_key` for the remote zone. Multiple `RemoteZoneSID`s can be defined in `server.config` to simultaneously federate with multiple zones.
+
+Information such as the remote zone's hostname and port are stored in the catalog in 3.x rather than in the server's configuration.
+
+As 3.x is built and run-in-place by the zone maintainer(s), the following diff can be applied to require SIDs be configured on the local server:
+```diff
```

Comment:
I'm actually not sure if this is necessary for getting this to work. It does give occasion to explain how the RemoteZoneSID should be structured, though. Might be a good idea to leave as an optional step (once I confirm that it is indeed optional). Thoughts?

Reply:
let's leave it, anything we/you learn is good to write down.